packages/ui/icons/Add.js

```diff
@@ -0,0 +1,118 @@
+function Add(props) {
```

Comment:
better rename add and search since these icons are not for general use, there will be conflicts with other general icons

Reply:
why are the icons not for general use, you can see here https://www.figma.com/file/oZ8bD7wkKYbGUxINEJqDtZ/Fluent-Wallet?node-id=21%3A970

Reply:
done